lib/kino/shorts.ex

```diff
@@ -0,0 +1,573 @@
+  end
+
+  @doc """
+  Renders and reads a new color input.
+
+  ## Options
+
+    * `:default` - the initial input value. Defaults to `#6583FF`
+  """
+  @spec read_color(String.t(), keyword()) :: String.t() | nil
+  def read_color(label, opts \\ []) when is_binary(label) and is_list(opts) do
+    Kino.Input.color(label, opts) |> Kino.render() |> Kino.Input.read()
+  end
+
+  @doc """
+  Renders and reads a new image input.
+
+  > #### Warning {: .warning}
+  >
+  > The image input is shared by default: once you upload a image,
+  > the image will be replicated to all users reading the notebook.
+  > Use `Kino.Control.form/2` if you want each user to have a distinct
+  > image upload with an explicit submission button.
+
+  ## Options
+
+    * `:format` - the format to read the image as, either of:
+
+      * `:rgb` (default) - the binary includes raw pixel values, each
+        encoded as a single byte in the HWC order. Such binary can be
+        directly converted to an `Nx` tensor, with no additional decoding
+
+      * `:png`
+
+      * `:jpeg` (or `:jpg`)
+
+    * `:size` - the size to fit the image into, given as `{height, width}`
+
+    * `:fit` - the strategy of fitting the image into `:size`, either of:
+
+      * `:contain` (default) - resizes the image, such that it fits in
+        a box of `:size`, but preserving the aspect ratio. The resulting
+        image can be smaller or equal to `:size`
+
+      * `:match` - resizes the image to `:size`, with no respect for
+        aspect ratio
+
+      * `:pad` - same as `:contain`, but pads the image to match `:size`
+        exactly
+
+      * `:crop` - resizes the image, such that one edge fits in `:size`
+        and the other overflows, then center-crops the image to match
+        `:size` exactly
+
+  """
+  @spec read_image(String.t(), keyword()) ::
+          %{
+            data: String.t(),
+            height: pos_integer(),
+            width: pos_integer(),
+            format: :rgb | :png | :jpeg
+          }
+          | nil
+  def read_image(label, opts \\ []) do
+    Kino.Input.image(label, opts) |> Kino.render() |> Kino.Input.read()
+  end
+
+  @doc """
+  Renders and reads a new audio input.
+
+  > #### Warning {: .warning}
+  >
+  > The audio input is shared by default: once you upload an audio,
+  > the audio will be replicated to all users reading the notebook.
+  > Use `Kino.Control.form/2` if you want each user to have a distinct
+  > audio upload with an explicit submission button.
+
+  ## Options
+
+    * `:format` - the format to read the audio as, either of:
+
+      * `:pcm_f32` (default) - the PCM (32-bit float) format. Note that
+        the binary uses native system endianness. Such binary can be
+        directly converted to an `Nx` tensor, with no additional decoding
+
+      * `:wav`
+
+    * `:sampling_rate` - the sampling rate (samples per second) of
+      the audio data. Defaults to `48_000`
+
+  """
```

Comment:
I'm concerned with duplicating all of the docs and options and keeping them in sync. Could we just point users towards the original docs?

Reply:
Yeah, I am torn. We want to make it accessible for terminal and autocompletion but in this case I think it is worth just pointing to the original docs. I will update later.